src/store/undoRedo/index.ts

```diff
@@ -115,15 +124,29 @@ export const actions: ActionTree<IUndoRedoState, IRootState> = {
       flows: rootState.flow,
     })
```

Comment:
would `newSnapshot` be a better name instead of `nextSnapshot`?
'next' is associated with the 'next in the stack' for me, whereas 'new' - is newly created, not placed anywhere yet

Reply:
I agree, `newSnapshot` would read better.

Reply:
renamed !